.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,8 @@
 <!--
+Please only send in a PR to branches which are currently supported: https://laravel.com/docs/5.8/releases#support-policy 
```

Comment:
```suggestion
Please only send in a PR to branches which are currently supported: https://laravel.com/docs/releases#support-policy 
```

Reply:
The docs don't support page forwards with anchor tags.